### ISSUE TITLE
Accessibility:List: Information about “number of items selected” is not conveyed to the user, immediately after selecting the list items. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3480]
Description: 

Case 1:
Information about “number of items selected” is not conveyed to the user, immediately after selecting the list items. 
Case 2:
Information about the selected list item is not conveyed, immediately after selecting the list item

Expected: information about selected item and number of items selected should be conveyed to the user, immediately after selecting the list items

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94648233-b5895080-030f-11eb-92be-2895eddaa9fc.png)

![image](https://user-images.githubusercontent.com/32538291/94648238-bae69b00-030f-11eb-91ff-4a84c3cd18dd.png)

![image](https://user-images.githubusercontent.com/32538291/94648243-bfab4f00-030f-11eb-9eb6-6ca016417a14.png)
